apps/console/src/features/users/components/guests/pages/guest-users-list.tsx

```diff
@@ -0,0 +1,475 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { hasRequiredScopes } from "@wso2is/core/helpers";
+import { AlertLevels, IdentifiableComponentInterface, TestableComponentInterface } from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import {
+    AnimatedAvatar,
+    AppAvatar,
+    ConfirmationModal,
+    DataTable,
+    EmptyPlaceholder,
+    LinkButton,
+    TableColumnInterface
+} from "@wso2is/react-components";
+import { AxiosError } from "axios";
+import React, { FunctionComponent, ReactElement, SyntheticEvent, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import { Dispatch } from "redux";
+import { Header, SemanticICONS } from "semantic-ui-react";
+import { deleteInvite, resendInvite } from "../../../../../extensions/components/users/api";
+import { InvitationStatus } from "../../../../../extensions/components/users/models";
+import { 
+    AppState, 
+    FeatureConfigInterface, 
+    UserListInterface, 
+    getEmptyPlaceholderIllustrations 
+} from "../../../../core";
+import { UserAccountTypesMain } from "../../../constants";
+import { deleteParentOrgInvite } from "../api/invite";
+import { UserInviteInterface } from "../models/invite";
+
+/**
+ * Props for the Guest users list component.
+ */
+interface GuestUsersListInterface extends IdentifiableComponentInterface, TestableComponentInterface {
```

Comment:
Do not use `TestableComponentInterface` in new components as it is depercated use `IdentifiableComponentInterface` instead.